infra/test/integration/go.mod

```
@@ -0,0 +1,76 @@
+module github.com/GoogleCloudPlatform/terraform-ecommerce-microservices-on-gke/test/integration
+
+go 1.19
```

FYI: The dev tools image currently contains GO 1.18, so this might cause an issue depending on the exact indirects.

```suggestion
go 1.18
```

---

Thanks for the heads-up! That helps. :)
I'll downgrade to 1.18.